### PR TITLE
combine parents

### PR DESCRIPTION
**Pull request by vshand11**
combine parents in release record.

### CODE REVIEW

**Review by n2iw — CHANGES_REQUESTED** [review]
Please add parent node updating based on relationship multiplicity (many-to-many etc.) support, thanks!

File: src/metadata_export.py [hunk 1]
```diff
@@ -353,6 +354,26 @@ def save_release(self, data_record, node_type, node_id, crdc_id):
...
+            return node_parents
+        if not node_parents or len(node_parents) == 0:
+            return release_parents
+        relationships = self.model.get_node_relationships(node_type)
+        if node_parents and len(node_parents):
+            for parent in node_parents:
+                relationship = relationships[parent["parentType"]]["dest_node"]
+                if not dict_exists_in_list(release_parents, parent, keys=["parentType", "parentIDPropName", "parentIDValue"]):
+                    if relationship == "many_to_many":
+                        release_parents.append(parent)
```
Comment (n2iw) [inline src/metadata_export.py:368]: Please make sure there will be only one copy of the same parent, if new data contains same parent(s) as previous ones.

**Review by n2iw — CHANGES_REQUESTED** [review]

File: src/metadata_export.py (hunk 1 above)
Comment (vshand11) [reply src/metadata_export.py:368]: If new data contains the same parent, the existing release record will not be changed.

**Review by n2iw — APPROVED** [review]
LGTM

**Pull request merged**